web/src/main/java/com/bazaarvoice/emodb/web/EmoService.java

```diff
@@ -147,9 +145,7 @@ public EmoService(String emoDdlPath, File configFile) {
     @Override
     public void initialize(Bootstrap<EmoConfiguration> bootstrap) {
         bootstrap.addCommand(new CreateKeyspacesCommand());
-        bootstrap.addCommand(new RegisterCassandraCommand());
         bootstrap.addCommand(new ListCassandraCommand());
```

Comment:
it probably makes sense to remove `ListCassandraCommand` also

Reply:
updated